docs/APP.md

```diff
@@ -1,4 +1,4 @@
-# App
+# aragonJS - App API
```

Comment:
Not sure if we should change the title to use aragonJS; see https://github.com/aragon/hack/pull/127#discussion_r271190596.

Maybe just "JavaScript — App API", etc. would work as headings?

Reply:
Shouldn’t we use “aragonAPI” instead of “App API”, since it is the main module?

We could have:

- aragonAPI − JavaScript
- aragonAPI − React
- aragonAPI − …

Or:

- aragonAPI for JavaScript
- aragonAPI for React
- aragonAPI for …

Or we could put it after, but it feels a bit weird IMO:

- JavaScript − aragonAPI
- React − aragonAPI
- … − aragonAPI

Reply:
```
aragonAPI for JavaScript
aragonAPI for React
aragonAPI for wrappers
aragonAPI for …
```

Sounds quite nice 👍

Reply:
I also like `aragonAPI for ...`. I'm a bit confuse still about the extent of the JavaScript implementation. I thought all `aragon.js` was the JavaScript implementation of `aragonAPI`.

Reply:
I understand the confusion. `aragon.js` is just the monorepo name. The suggestions above indicate the actual semantic meaning of each part of it.

Reply:
This discussion got me thinking. Now when I think of `@aragon/wrapper` it's actually just the Client/Wrapper API in the sense that it provides an API for clients, e.g. CLI or the official client (`aragon/aragon`).

Reply:
Following a conversation with @0xGabi I think it's important we indicate three things:
- It's the App API
- It's part of the Javascript implementation of aragonAPI
 
Given the other page with the explanation about the JavaScript explanation I think `aragonAPI for Apps` should be fine.